proposals/2025/Add Loop.md

```diff
@@ -0,0 +1,336 @@
+### ADD
+* `Plant`/`HeatingPlant` (or only under /`Boiler`? Or also /`DistricHeating`?)
+    * `FluidLoop`
+    * May be duplicated with HeatingPlant/Boiler/Some flow rate values
+* `Plant`/`CoolingPlant` (or only under /`Chiller`? Or also /`DistrictChilledWater`?)
+    * `FluidLoop`
+* `Plant`/`CondenserPlant`/`WaterCooled`
+    * `FluidLoop`
+* `DomesticHotWaterSystem`/`Recirculation`
+    * `FluidLoop` (`HotWaterLoop`)
+    * If /`HotWaterDistributionType` is "Looped"
+* `HeatRecoverySystem`
+    * `FluidLoop`
+### USE REF (linking element)
+* `HeatingAndCoolingSystems`
+    * `HeatingSource`/`HeatingSourceType`/`SourceHeatingPlantID`: linked to the plant loop above 
+    * `CoolingSource`/`CoolingSourceType`/`CoolingPlantID`: linked to the plant loop above
+* `PumpSystem`/`LinkedSystemIDs`/`LinkedSystemID`
+    * Loop for which the pump system serves
+### Global element structure
+Considering the shared properties/definitions of `FluidLoop`, ideal implementation of the addition is through adding global element `FluidLoop` and call/refer to it under every elements above to call the usage of `FluidLoop`
+### Child elements
+Some of the child properties under `loop` in 229P are already defined (in some other way) within BuildingSync, we propose to add these new childs for the added `FluidLoop`:
+* `ID`
+* `Type`: "HEATING","COOLING","HEATING_AND_COOLING","CONDENSER", "OTHER"
+* `PumpPowerPerFlowRate`: W-s/L
+* `LinkedChildLoops`/`LinkedChildLoop`
+* `FluidLoopDesignAndControl`
+    * `DesignSupplyTemperature`
+    * `DesignReturnTemperature`
+    * `MinimumFlowFraction`
+    * `operation` -> `FluidLoopOperationOptions`
+    * `operation_schedule`
+    * `flow_control` -> `FluidLoopFlowControlOptions`
+    * `temperature_reset_type`
+        * `outdoor_high_for_loop_supply_reset_temperature`
+        * `outdoor_low_for_loop_supply_reset_temperature`
+        * `loop_supply_temperature_at_outdoor_high`
+        * `loop_supply_temperature_at_outdoor_low`
+        * `loop_supply_temperature_at_low_load`
+Other fields: `is_sized_using_coincident_load` is too too specific to modeling; `has_integrated_waterside_economizer` is covered by existing `WaterSideEconomizer` element under `CondenserPlant`.
+
+## Implementation
+Global definition
+```xml
+    <xs:element name="FluidLoops">
+        <xs:annotation>
+            <xs:documentation>List of fluid loops for HeatingSource.</xs:documentation>
+        </xs:annotation>
+        <xs:complexType>
+            <xs:sequence>
+                <xs:element name="FluidLoop" type="auc:FluidLoopType" maxOccurs="unbounded">
+                    <xs:annotation>
+                        <xs:documentation>fluid loop for HeatingSource.</xs:documentation>
+                    </xs:annotation>
+                </xs:element>
+            </xs:sequence>
+        </xs:complexType>
+    </xs:element>
+    <xs:complexType name="FluidLoopType">
```

Comment:
Should this type have child elements for year installed, year of manufacture, primary fuel, manufacturer, model number, location, linked premises, quantity, equipment ID, etc.?